cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/opencv-control.tsx

```diff
@@ -254,6 +343,36 @@ class OpenCVControlComponent extends React.PureComponent<Props & DispatchToProps
         );
     }
 
+    private renderImageContent():JSX.Element {
+        return (
+            <Row justify='start'>
+                <Col>
+                    <CVATTooltip title='Histogram equalization' className='cvat-opencv-image-tool'>
+                        <Button
+                            className={this.imageModifier('histogram') ? 'cvat-opencv-image-tool-active' : ''}
+                            onClick={(e: React.MouseEvent<HTMLElement>) => {
+                                const modifier = this.imageModifier('histogram');
+                                if (!modifier) {
+                                    this.enableImageModifier(openCVWrapper.imgproc.hist(), 'histogram');
+                                } else {
+                                    const button = e.target as HTMLElement;
+                                    button.blur();
+                                    this.disableImageModifier('histogram');
+                                    const { changeFrame } = this.props;
+                                    const { frame, canvasInstance } = this.props;
+                                    canvasInstance.configure({ forceFrameUpdate: true });
+                                    changeFrame(frame, false, 1, true);
```

Comment:
Where canvasInstance.configure({ forceFrameUpdate: false }); for this case is located?

Reply:
As it was discussed, we cant configure `forceFrameUpdate` flag in this function because we are dispatching `CHANGE_FRAME` action. So flag will be disabled on `canvas.setup` event. (I'll add try/finally block to event handler, so force update flag will be disabled right after event was fired)

Reply:
Ok